nsxt/resource_nsxt_policy_security_policy_test.go

```diff
@@ -240,7 +350,16 @@ func testAccNsxtPolicySecurityPolicyExists(resourceName string, domainName strin
 			return fmt.Errorf("Policy SecurityPolicy resource ID not set in resources")
 		}
 
-		_, err := nsxClient.Get(domainName, resourceID)
+		var err error
+		if isPolicyGlobalManager(testAccProvider.Meta()) {
```

Comment:
can we use resourceNsxtPolicySecurityPolicyExistsInDomain here?

Reply:
Thanks for the catch. Yes we can, but that way we will lose track of the error message here. Instead the error will be logged in logAPIError in resourceNsxtPolicySecurityPolicyExistsInDomain. Do you think it's OK here?

Reply:
Yes, I think its OK since we don't expect errors other than 404 at this stage

Reply:
Sure. Let me fix this here. Thanks!